cli/src/commands/content-directory/updateEntityPropertyValues.ts

```diff
@@ -0,0 +1,174 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import { flags } from '@oclif/command'
+import { Class as ContentDirectoryClass } from '@joystream/types/content-directory'
+import inquirer from 'inquirer'
+import { CLIError } from '@oclif/errors'
+import { InputParser } from '@joystream/cd-schemas'
+
+const CONTEXTS = ['Member', 'Curator', 'Lead'] as const
+
+type CreateEntityArgs = {
+  id: string
+}
+
+type CreateEntityFlags = {
+  context: typeof CONTEXTS[number]
+}
+
+type Question = {
+  name: string
+  message: string
+  type: 'list' | 'input'
+  choices?: string[]
+  filter?: (answer: string) => string | null | number | boolean
+}
+
+export default class UpdateEntityPropertyValues extends ContentDirectoryCommandBase {
```

Comment:
As far as permissions are concerned, here we would need to be careful about class properites that have `is_locked_from_maintainer` and `is_locked_from_contoller` flags set.

If we just prompt for values for all class properties and provide them in the final extrinsic, we may get an unavoidable runtime error in case the actor has no privilages to set value for given property (ie. `Channel.isCensored`)